docs/test-vectors/vLEIIdentityCredential.json

```diff
@@ -0,0 +1,43 @@
+{
+  "good": [
+    {
+      "vLEIIdentityCredential": "Ullam veritatis vel.",
+      "type": "vLEIIdentityCredential",
```

Comment:
It would be helpful to see an example of a LEI Code here

Reply:
done, vLEIIdentityCredential property was renamed with leiCode and it contains  LEI code (20 characters alphanumeric string)